### PR TITLE
tls: minor improvements

### PR DESCRIPTION
**Pull request by rst0git**
This pull request contains minor improvements to the code in `tls.c`.

### CODE REVIEW

**Comment by avagin** [comment]
lgtm

File: criu/tls.c [hunk 1]
```diff
@@ -40,20 +40,24 @@ void tls_terminate_session(void)
 
 	if (session) {
 		do {
-			/* don't wait for peer to close connection */
+			/* Initiate a connection shutdown but don't
```
Comment (Snorch) [inline criu/tls.c:43]: nit: multi-line comment code stile:
```
/*
 * line 1
 * line 2
 */
```

**Review by Snorch — APPROVED** [review]
Looks good.

File: criu/tls.c (hunk 1 above)
Comment (rst0git) [reply criu/tls.c]: Thanks, fixed.

**Comment by adrianreber** [comment]
Please rebase.

**Pull request merged**